stac_check/lint.py

```diff
@@ -33,6 +36,36 @@ def __post_init__(self):
         self.file_name = os.path.basename(self.item).split('.')[0]
         self.best_practices_msg = self.create_best_practices_msg()
 
+    def parse_config(self, config_file):
+        default_config = {
+            'linting': {
+                'searchable_identifiers': True, 
+                'percent_encoded': True,
+                'item_id_file_name': True,
+                'catalog_id_file_name': True,
+                'check_summaries': True,
+                'null_datetime': True,
+                'check_unlocated': True,
+                'check_geometry': True,
+                'bloated_links': True,
+                'bloated_metadata': True,
+                'check_thumbnail': True,
+                'links_title': True,
+                'links_self': True,
+            }
+        }
+        if config_file == False:
+            config_file = "stac-check.config.yml"
```

Comment:
Would it make sense to allow for an environment variable to point to the config file as well? E.g. `STAC_CHECK_CONFIG` or something?